### DISCUSSION TITLE
[ideas] Please add command to unassign security role of user for the environment

### DISCUSSION THREAD

**Discussion by sanjeev40084** [microsoft/powerplatform-vscode#680] [Ideas]
Currently we have command to assign security role for user, please also add command to unassign security role. We are admin for about 50 power platform environments and all my team members have SA role. One of my team member is moving to new team and his SA role needs to be removed. Currently we need to do one environment at a time :( 

`pac admin unassign-user -env [env-id] -u $upn -r [role-name]`

I think the best approach will be to introduce new user attribute and have flag to assign and unassign roles something like this:

To assign role:
`pac admin user -env [env-id] -u $upn -r [role-name] -assign`

To unassign role:
`pac admin user -env [env-id] -u $upn -r [role-name] -unassign`

**Comment by Hjaf** [comment]
I don't understand why this is not addressed, I struggle to make sense of why there is a way to assign a user but not revoke with pac cli.
